api/data_explorer/controllers/export_url_controller.py

```diff
@@ -165,7 +165,7 @@ def _get_range_clause(column, value):
 
 
 def _get_clause(column, type, value):
-    """Returns a single condition of a WHERE clause, eg "gender = female"."""
+    """Returns a single condition of a WHERE clause, eg "gender = female, male"."""
```

Comment:
Hmm, selecting both genders isn't realistic. Also, there should be ORs and ANDs. How about
```
"((age76 >= 20 AND age76 < 30) OR (age76 >= 30 AND age76 < 40))"
```